src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Quotas & Events'
+sidebar_order: 0
+---
+
+Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events to be tracked. This number of events is your quota. When an event is accepted by Sentry, it counts toward your quota.
+
+Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.
+
+## Key Terms
+
+Let’s clarify a few terms to start:
+-   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a unique set of characteristics, called its fingerprint.
```

Comment:
I'm realizing that the description of the "fingerprint" here is not correct. The only unique characteristic of each event is its `event_id`. The fingerprint is not unique, in contrast, as you explain below.

Reply:
Thank you. I've removed "unique" here.